cc/oak_echo_raw_enclave_app/main.cc

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <unistd.h>
+
+constexpr int CHANNEL_FD = 10;
```

Comment:
Can we have an `oakio.h` file that defines these constands?

Reply:
Yes, we'll need a better support library for all of it. But yet again: baby steps and I'd like things to be as self-contained and simple as possible.